### DISCUSSION TITLE
DEV1210G - Collaborating effectively using GitHub

### DISCUSSION THREAD

**Discussion by rajbos** [community/community#73680] [Other Feature Feedback, Questions, & Ideas]
## Session abstract

A lot of teams work on GitHub and use only a small part of the available features. In this session, Rob will show you how to use GitHub to handle team processes like on-boarding a new team member (while skipping the boring text documents or wiki pages), setting up daily scrums, and even sending out updates of team accomplishments at the end of your sprint. We'll explain how to set up an effective team environment using GitHub, with demos from open source repositories that'll help you get started right away!

## Speakers
|Avatar|Name|
|---|---|
![rajbos]| Rob Bos |

Blog: https://devopsjournal.io
LinkedIn: https://linkedin.com/in/bosrob

[rajbos]: https://avatars.githubusercontent.com/rajbos?s=80

Feel free to drop you questions below!

**Comment by oseijoel6111** [comment]
that will be great oseijoel6111@gmail.com
